README.md

```diff
@@ -16,8 +16,13 @@ So far we tested Northstar on
 * 64-bit ARM
 * x86_64
 
-### Missing from implementation
+### Everything that is checked is implemented
 
+- [x] On-the-fly verification of container content
```

Comment:
Transparent integrity checks backed by [dm-verity](https://www.kernel.org/doc/html/latest/admin-guide/device-mapper/verity.html)

Reply:
dm-verity is what we use...it's not the feature